server.js

```diff
@@ -40,7 +42,7 @@ if (app.get("env") !== "dev") {
   try {
     await HIBP.loadBreachesIntoApp(app);
   } catch (error) {
-    console.error(error);
+    log.error(error);
```

Comment:
Same as earlier comment... "Above we did something like log.error("generating", {err: err});"

Not sure if we need a "subject" here.